gdpr/impl_test.go

```diff
@@ -23,8 +23,8 @@ func TestNoConsentButAllowByDefault(t *testing.T) {
 		},
```

Comment:
I made minimal changes to `impl_test.go`. My focus was on the vendor list fetcher.